src/telemetry.ts

```diff
@@ -4,9 +4,23 @@ import { ErrorCodes } from 'types'
 // Set at build time
 declare const worldIdJSVersion: string
 
+const posthogFetchError = { name: 'posthog-error' }
+
+window.onunhandledrejection = function (event) {
+  return event.reason !== posthogFetchError
+}
+
+async function posthogFetch(input: RequestInfo, init?: RequestInit) {
+  try {
+    return await window.fetch(input, init)
+  } catch (error) {
+    throw posthogFetchError
```

Comment:
@michaelfurutin Is it possible to pass error to posthogFetchError like `throw posthogFetchError(error)`?

Reply:
@igorosip0v to keep the original error? done: https://github.com/worldcoin/world-id-js/pull/29/commits/6ff4aa2eb3ed16106c7dec36d8caacb1d786b655